### ISSUE TITLE
nvda does not read microsoft teams when hovering mouse

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13616]
### Steps to reproduce:
where I work I'm helping a user who needs to use microsoft teams.
The user switches between using the mouse and the keyboard.
I'm totally blind so maybe details are missing.
Open microsoft teams, now hover over the teams interface mainly
toolbar where you can choose between calendar, chats, teams and more.
Also hover over other areas of microsoft teams.
### Actual behavior:
nvda is silent, not reading the teams interface.
### Expected behavior:
when hovering over mouse the teams interface nvda should read what's under the mouse, just like it does in other programs.
Navigation using the keyboard is normal.
#### NVDA installed/portable/running from source:
portable
#### NVDA version:
nvda.exe, NVDA alpha-25209,21367c8e

#### Windows version:
10 10.0.19044.1645
#### Name and version of other software in use when reproducing the issue:
Teams.exe, Microsoft Teams 1.5.00.9163
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by feerrenrut** [comment]
Related issues #11983 #10650 and #11066

**Comment by Qchristensen** [comment]
I just tested this for another user and I can confirm:
a) The issue is still present in Teams version 1.0 with NVDA 2022.1
b) The issue is also present testing with Narrator, so is possibly a Teams issue, however also:
c) The buttons are all correctly identified when you press tab to get to the controls and then arrow through them.

**Comment by Adriani90** [comment]
Yes the issue is definitely in Teams and needs eternal fix, probably from Chromium side as well. It seems the information about the labels of buttons is not exposed when hovering with the mouse.
cc: @msftedad can you please take this up with the development team and get back to us with an update on this issue?

**Comment by msftedad** [comment]
@Adriani90 - we are Checking on this issue, will let you know the updates here.

**Comment by msftedad** [comment]
@Adriani90 - We have forwarded this issue to MS Teams. Thank you!

**Comment by Adriani90** [comment]
I also created
https://issues.chromium.org/issues/380416882
Feel free to add constructive input there, or complement what I've described in case there is something that I missed.

@msftedad this issue might also be a general one with the Chromium rendering engine, so feel free to engage your development team to work on this together with Chromium developers. I guess this is a broader issue than I think.

**Comment by Adriani90** [comment]
@msftedad can you please report this to the development team? is there any update?

**Comment by msftedad** [comment]
@Adriani90, We have forwarded this issue to development team. We will provide you with the tracking id once we receive update from them. Thanks
